src/containers/StructurePage/folderComponents/SettingsMenu.tsx

```diff
@@ -44,12 +44,12 @@ const SettingsMenu = ({ node, rootNodeId, onCurrentNodeChanged, nodeChildren }:
     <DialogRoot position="top">
       <DialogTrigger asChild>
         <IconButton
-          variant="secondary"
+          variant="clear"
```

Comment:
Passererer dette kontrastkrav?

Reply:
Jepp

Reply:
Men lurer på om jeg skal bytte ut ikonet med `MoreFill` for å gjøre det tydeligere!